api.swagger.yaml

```diff
@@ -2872,6 +3312,10 @@ components:
           type: string
           format: uuid
           example: 01234567-89ab-cdef-1234-56789abcdef0
+        has_children:
```

Comment:
Why are we changing existing endpoints ?

Reply:
Not a breaking change, should be fine